File: helloworld/app/src/main/java/com/example/helloworld/MainActivity.java
package com.example.helloworld;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    Button btn;
    TextView tv;
    Button logout_button;
    ImageView img;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        btn= findViewById(R.id.clickme);
        tv= findViewById(R.id.text1);
        img= findViewById(R.id.img);
        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                tv.setText("Welcome Ankit");
                img.setImageResource(R.drawable.aaaa);
                tv.setTextColor(getResources().getColor(R.color.colorAccent));
                Toast.makeText(getApplicationContext(),"you are logged in as ankit",Toast.LENGTH_SHORT).show();
            }
        });
        logout_button = findViewById(R.id.logoutbutton);
        logout_button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                tv.setText("Please login");
                tv.setTextColor(getResources().getColor(R.color.colorAccent));
                img.setImageResource(R.drawable.initial_background);
                Toast.makeText(getApplicationContext(),"logged out successfully",Toast.LENGTH_SHORT).show();
            }
        });



    }
}

File: Dice/app/src/main/java/com/example/dice/MainActivity.kt
package com.example.dice

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.Button
import android.widget.ImageView
import android.widget.Switch

import java.util.*


class MainActivity : AppCompatActivity() {
    lateinit var diceimage: ImageView
    lateinit var rollButton: Button
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        diceimage = findViewById(R.id.dice_image)
        rollButton = findViewById(R.id.roll_button)
        rollButton.setOnClickListener {
            val rnum = Random().nextInt(6) + 1
            //Log.v("rnum is =" , rnum.toString())
            when(rnum){
                1-> diceimage.setImageResource(R.drawable.dice_1)
                2-> diceimage.setImageResource(R.drawable.dice_2)
                3-> diceimage.setImageResource(R.drawable.dice_3)
                4-> diceimage.setImageResource(R.drawable.dice_4)
                5-> diceimage.setImageResource(R.drawable.dice_5)
                6-> diceimage.setImageResource(R.drawable.dice_6)
            }
        }
    }
}
